packages/default-theme/__e2e__/integration/checkout_desktop.spec.js

```diff
@@ -0,0 +1,38 @@
+describe('Test Checkout funcionality Desktop view' , () => {
+    
+    beforeEach(() => {
+       cy.viewport(1280,800)
+       // run these tests as if in a desktop
+       // browser with a 800p monitor
+       
+    })
+    describe('Add product to the cart-product is in the cart', () => {
+        it('Navigate to Shopware-pwa site', () => {
+            cy.visit('https://shopware-pwa.storefrontcloud.io/')
```

Comment:
@akrajna Add https://shopware-pwa.storefrontcloud.io/ as baseUrl in your configuration file (cypress.json) then just use cy.visit('/')